packages/bot-web-ui/src/components/dashboard/dashboard-component/__tests__/dashboard-components.spec.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+// import Dashboard from '../../dashboard';
```

Comment:
@rupato-deriv probably can we remove it?

Reply:
done ✅